packages/next-document/README.md

```diff
@@ -0,0 +1,98 @@
+<!-- Title -->
+<h1 align="center">
+👋 Welcome to <br/><code>@expo/next-document</code>
+</h1>
+
+<!-- Header -->
+
+<p align="center">
+    <b>The document component for Next.js projects using Unimodules</b>
+    <br/>
+    <br/>
+    <a aria-label="Circle CI" href="https://circleci.com/gh/expo/expo-cli/tree/master">
+        <img alt="Circle CI" src="https://flat.badgen.net/circleci/github/expo/expo-cli?label=Circle%20CI&labelColor=555555&icon=circleci">
+    </a>
+</p>
+
+---
+
+## 🏁 Setup
+
+Install `@expo/next-document` in your project.
+
+```sh
+yarn add @expo/next-document
+```
+
+## ⚽️ Usage
+
+Re-export this component from the `pages/_document.js` file of your Next.js project.
+
+```js
+// pages/_document.js
+import Document from '@expo/next-document';
```

Comment:
Since the Next.js server that's currently hard-coded in XDL is strongly coupled with Next.js (even if `next` is not a direct dependency of XDL), maybe we should already plan to extract it out as well? Then a more appropriate name for this package might be something like `@expo/next-plugin`?
```suggestion
import { Document } from '@expo/next-plugin';
```